Package.swift

```diff
@@ -16,6 +16,7 @@ import Foundation
 // Therefore, it's important that we only expose modules that we want customers to use. Internal modules should not be included in this array.
 var products: [PackageDescription.Product] = [
     .library(name: "Tracking", targets: ["CioTracking"]),
+    .library(name: "DataPipeline", targets: ["CioDataPipeline"]),
```

Comment:
```suggestion
    .library(name: "DataPipeline", targets: ["CioDataPipelines"]),
```

Somewhat nit-picky, but important to me since it's customer facing. 

Since [we call the product "Data Pipelines"](https://customer.io/data-pipelines/) I think it makes sense for the public SDK module name to match that naming.